easybuild/easyconfigs/s/SDL2_mixer/SDL2_mixer-2.8.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
```

Comment:
I assume this was based on the easyconfig of the previous version, could you please also readd the author comment at the top?
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/s/SDL2_mixer/SDL2_mixer-2.6.3-GCCcore-11.3.0.eb#L1